frontend/src/Form/beta/ModelCardForm.tsx

```diff
@@ -0,0 +1,120 @@
+import { Box, Divider, Stack, StepButton, StepLabel, Stepper } from '@mui/material'
+import MaterialStep from '@mui/material/Step'
+import { useTheme } from '@mui/material/styles'
+import Form from '@rjsf/mui'
+import { DescriptionFieldProps } from '@rjsf/utils'
+import validator from '@rjsf/validator-ajv8'
+import React, { Dispatch, SetStateAction, useState } from 'react'
+import CustomTextInput from 'src/MuiForms/CustomTextInput'
+import TagSelector from 'src/MuiForms/TagSelectorBeta'
+import { setStepState } from 'utils/beta/formUtils'
+
+import { SplitSchemaNoRender } from '../../../types/interfaces'
+import Nothing from '../../MuiForms/Nothing'
+
+export default function ModelCardForm({
+  splitSchema,
+  setSplitSchema,
+  canEdit = false,
+}: {
+  splitSchema: SplitSchemaNoRender
+  setSplitSchema: Dispatch<SetStateAction<SplitSchemaNoRender>>
+  canEdit?: boolean
+}) {
+  const [activeStep, setActiveStep] = useState(0)
+  // TODO - add validation
```

Comment:
Not something I'm going to try to force through, but I'm a big fan of linking TODOs to tickets that have been created to capture the required changes. This comment especially sounds like something we definitely don't want to miss.